### DISCUSSION TITLE
Is it possible to disable console redrawing in Parcel v2 (beta)

### DISCUSSION THREAD

**Discussion by RPDeshaies** [parcel-bundler/parcel#4984] [General]
Hey,

I'm currently looking into upgrading a project to Parcel V2 (in preparation for the official release). 

This project uses `lerna` as well `concurrently` to run multiple packages and tasks at the same time. 

When using v2 of Parcel, starting our project ends up making the terminal very "verbose/noisy" since Parcel shows which files it's building and (I think) redraws the terminal every time there's a new file to build. 

It seems like Lerna or concurrently wont handle this since tasks could be executed in parallel and it would not know "which line to redraw" (which makes sense).

I was wondering if there's a way to make Parcel only display two messages (like in v1 I think). 

Something like  "🚧 Building" and "✨  Built"

Otherwise, would you guys have any recommendations for alternative ways of handling this issue ?

Discussion on lerna I found that oriented me into thinking what is the underlying issue: https://github.com/lerna/lerna/issues/1265#issuecomment-367050336

![image](https://user-images.githubusercontent.com/6224111/89534599-3241f780-d7c3-11ea-8603-ef4738c29c7d.png)

**Comment by mischnic** [comment]
There is `--log-level warn`, though that doesn't print anything unless some warning/error occurs...

**Reply by RPDeshaies** [reply]
Yeah I thought about that too and was thinking "yeah it's less verbose, but a bit too silent" 😅 

I'm guessing any parcel task ran within a parallel context might encounter this "issue" (I mean it's not the end of the world, just a bit unpleasant).

So using things like lerna, concurrently, yarn workspaces, bolt and what not becomes a less pleasant experience.

Is there a way to know when a task is running in a parallel context (which would make console redraw useless) so that the logs a different for that use case ?

**Reply by mischnic** [reply]
Personally, I don't find these "Building thing.js" messages useful anyway...
Maybe we should have another log-level that does what you described (between warn & info) that might even be the default?

> Is there a way to know when a task is running in a parallel context (which would make console redraw useless) so that the logs a different for that use case ?

No

**Reply by RPDeshaies** [reply]
> Personally, I don't find these "Building thing.js" messages useful anyway.

They are interactive and fun to look at but apart from that, don't provide much else I agree

> Maybe we should have another log-level

That could be interesting indeed! I initially wanted to propose having the `info` level be like what I proposed and leave the "Building thing.js" live inside the `verbose` level but I don't know fully the context of "what should be or not be inside the verbose log level".

I feel like there could also be (minor) performance boost at logging fewer things in the console. 

Perhaps the levels could be like "error, warn, info, ci, verbose" where `ci` logs every file built while `info` is the default.

I'm thinking CI because in a continuous integration process, perhaps you want to see "everything that is happening"? Mostly thinking out loud here!
